src/main/java/no/nav/foreldrepenger/oversikt/innhenting/inntektsmelding/FpSakInntektsmeldingDto.java

```diff
@@ -0,0 +1,33 @@
+package no.nav.foreldrepenger.oversikt.innhenting.inntektsmelding;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.List;
+
+public record FpSakInntektsmeldingDto(
+    boolean erAktiv,
+    BigDecimal stillingsprosent,
+    BigDecimal inntektPrMnd,
+                              BigDecimal refusjonPrMnd,
+                          String arbeidsgiverNavn,
+                          String journalpostId,
+                              String kontaktpersonNavn,
```

Comment:
navn og personnummer for kontaktperson, er dette noe bruker skal få se? Virker litt iffy å lagre personopplysninger om en annen part løst her